packages/nodes-base/nodes/QuestDb/QuestDb.node.ts

```diff
@@ -73,6 +75,34 @@ export class QuestDb implements INodeType {
 				required: true,
 				description: 'The SQL query to execute.',
 			},
+			{
+				displayName: 'Use query parameters',
+				name: 'useQueryParams',
+				type: 'boolean',
+				displayOptions: {
+					show: {
+						operation: ['executeQuery'],
+					},
+				},
+				default: false,
+				required: true,
+				description: 'Use Parametrized Queries, where variables are replaced using $1, $2, etc.. <br>Do not use this for regular n8n expressions.',
+			},
+			{
+				displayName: 'Properties',
+				name: 'properties',
+				type: 'string',
+				displayOptions: {
+					show: {
+						operation: ['executeQuery'],
+						useQueryParams: [true],
+					},
+				},
+				default: '',
+				placeholder: 'qty,price',
```

Comment:
ref comment in `Postgres.node.ts`

would also update the query placeholder accordingly 👍🏼 

```suggestion
				placeholder: 'quantity,price',
```